src/views/ScenesReducer.js

```diff
@@ -73,6 +73,16 @@ function areRoutesShallowEqual(
   return shallowEqual(one, two);
 }
 
+export function reduxToComponentStateHelper(nextState: NavigationState) {
```

Comment:
This really isn't being used since we are going to isPurged route instead of deleting items off the stack.

Reply:
@sdg9 done